keyboards/dichotomy/config.h

```diff
@@ -52,6 +52,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *  These options are also useful to firmware size reduction.
  */
 
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This should be removed, as it's no longer used (since it's the default behavior now).